packages/gatsby-tinacms-git/package.json

```diff
@@ -35,6 +35,7 @@
     "docs": "npx typedoc"
   },
   "dependencies": {
+    "tinacms": "^0.16.0",
```

Comment:
This should instead by a `peerDependency` and a `devDependency`. See [gatsby-tinacms-json](https://github.com/tinacms/tinacms/blob/master/packages/gatsby-tinacms-json/package.json#L34-L43) for an example.